### ISSUE TITLE
Object no align

### ISSUE THREAD

**Issue by iLenTheme** [salkar/morrigan-jquery-editor#3]
The images and videos only inserted to the left of the editor ?, no way I can center? I think a good editor is that you can improvements

**Comment by salkar** [comment]
Yes, video and image blocks inserts only to left side. It is done because of a broken cross-browser text  floating of pictures on both sides of text (especially in IE). 
Since the main advantage of this editor is predictable behavior, I removed the possibility of both sides blocks floating, because in some cases it worked unpredictable and broken. In addition, browsers are ill-suited for the polygraphic layout now. 
Later I plan to make an possibility to center the block without text wrapping and the ability to add to the block a couple of videos or pictures.

**Comment by iLenTheme** [comment]
It is only for ruby?

**Comment by salkar** [comment]
No, it is suitable for any server-side language. Server needed only for image uploading here.

**Comment by iLenTheme** [comment]
was ready, I hope you can make improvements :D
